src/deluge/gui/menu_item/arpeggiator/octaves.h

```diff
@@ -29,5 +29,8 @@ class Octaves final : public Integer {
 	bool isRelevant(ModControllableAudio* modControllable, int32_t whichThing) override {
 		return !soundEditor.editingGateDrumRow();
 	}
+	void getColumnLabel(StringBuf& label) override {
+		label.append(deluge::l10n::getView(deluge::l10n::built_in::seven_segment, this->name).data());
```

Comment:
clang-tidy is correct, please use `l10n::get` here instead of `l10n::getView().data()`
```suggestion
		label.append(deluge::l10n::get(deluge::l10n::built_in::seven_segment, this->name));
```

Reply:
done